### DISCUSSION TITLE
Option for spaces inside parentheses?

### DISCUSSION THREAD

**Discussion by taupecat** [sibiraj-s/vscode-scss-formatter#7] [Feature request] [answered]
Can an option be added to allow for spaces at the beginning and end of parentheses, in the WordPress PHP style? I do this to keep my code consistent between PHP, JS, and SCSS. For example:

Instead of this: `@media (min-width: 48em) {`,

This: `@media ( min-width: 48em ) {`

Thanks.

**Comment by sibiraj-s** [comment, marked as answer]
The formatter relies on prettier for formatting, and prettier doesn't have such option. And highly doubt that they won't be adding this now. So it is not possible to do that.

**Comment by taupecat** [comment]
Ah, gotcha. Thanks anyway.
